.gitignore

```diff
@@ -21,4 +21,4 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 public/index.html~HEAD
-package-lock.json
```

Comment:
Why did you remove the package-lock.json from .gitignore? This file changes for almost everyone and people will have to deal with many merge conflicts and a large diff file like [below](https://github.com/openmrs/openmrs-ocl-client/pull/2/files#diff-32607347f8126e6534ebc7ebaec4853d)

Reply:
@flacode I removed it because it is the best practice given the kind of installation we did, It is helpful in case someone clones this repo and has to install the dependencies. It has the exact versions which avoids errors for anyone setting up the project. Refer to this for more info. [Why package-lock](https://docs.npmjs.com/files/package-lock.json)

Reply:
@hadijahkyampeire, the package.json file keeps the versions of these dependencies. The package-lock.json file is an auto generated file that you have no control of how it changes. Therefore, there is no need of resolving merge conflicts in an auto generated file.

Reply:
@hadijahkyampeire, the package.json file keeps the versions of these dependencies. The package-lock.json file is an auto generated file that you have no control of how it changes. Therefore, there is no need of resolving merge conflicts in an auto generated file.